src/services/balances.rs

```diff
@@ -38,12 +38,14 @@ pub async fn balances(
         .await
         .unwrap_or(0.0);
 
+    let native_currency = info_provider.chain_info().await?.native_currency;
```

Comment:
Maybe a default should be provided here wdyt? 🤔

Reply:
I would say yes. As currently and error in the response for this request would result in the gw itself propagating the error invalidating the rest of the data. Not sure what a sensible default would be. 

We could either use
```
#NATIVE_COIN_DECIMALS=18
#NATIVE_COIN_SYMBOL=ETH
#NATIVE_COIN_NAME=Ether
```

Or replace `ETH` and `Ether` with `NC` and `Native Coind` respectively (not so happy about this as something called `NC` may already exist :thinking: )

Reply:
Added Ether as default value 5630708